README.md

```diff
@@ -197,6 +245,14 @@ auto-cpufreq daemon and its systemd service, along with all its persistent chang
 
 `sudo auto-cpufreq --remove`
 
+You can also stop the service with:
```

Comment:
No point to have lines 248 to 255, `sudo auto-cpufreq --remove` and/or `sudo ./auto-cpufreq-installer --remove` will do this for you automatically. If done manually it can only break things, so let's leave these lines out.